src/Npgsql/MetricsReporter.cs

```diff
@@ -107,9 +104,10 @@ static MetricsReporter()
             unit: "{connection}",
             description: "The maximum number of open connections allowed.");
 
-        PreparedRatio = Meter.CreateObservableUpDownCounter(
+        PreparedRatio = Meter.CreateObservableGauge(
             "db.client.commands.prepared_ratio",
             GetPreparedCommandsRatio,
+            unit: "s",
```

Comment:
What exactly does `s` mean here?

Reply:
`s` means seconds, but I don't think it makes sense here.

Reply:
Yep, sorry - this was a botched cleanup. Will remove the unit as before.